Applications/SlicerApp/Testing/Cpp/qSlicerModuleFactoryManagerTest1.cxx

```diff
@@ -48,7 +48,7 @@ int qSlicerModuleFactoryManagerTest1(int argc, char * argv[])
 
   qSlicerAbstractCoreModule * abstractModule =
     moduleFactoryManager.moduleInstance(moduleName);
-  if( abstractModule == NULL )
```

Comment:
Considering that this code is compiled, should the update be associated with the first commit ?

Reply:
It will, but I didn't compile the tests, so clang-tidy ignored them. Will update later.